lib/lighthouse/benefits_claims/service.rb

```diff
@@ -168,12 +165,16 @@ def validate526(body,
     private
 
     def build_request_body(body)
-      {
-        data: {
-          type: 'form/526',
-          attributes: body
+      body = body.as_json
+      if body.dig('data', 'attributes').nil?
+        body = {
+          data: {
+            type: 'form/526',
+            attributes: body
+          }
```

Comment:
Could append `.as_json` to this variable assignment and remove the redundancy on L177.